bundles/AdminBundle/Resources/public/js/pimcore/document/edit.js

```diff
@@ -404,6 +404,34 @@ pimcore.document.edit = Class.create({
         catch (e2) {
         }
 
+        return values;
+    },
+
+    getRequiredEditableValues: function () {
+        var values = {};
```

Comment:
I think it's sufficient to use a simple array here for `values` and just return the names of the empty required editables: 

```js
var values = [];
// ...
values.push(editableName); 
// ...
```